LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright 2025 Datadog, Inc.
```

Comment:
You're not really meant to change this here. These are instructions on how to copy this text to apply the License to specific files.

Reply:
Updated here: https://github.com/DataDog/datadog-serverless-compat-dotnet/commit/5e6dd401e76cdd77c2c328a20c58e3ad980a2994